internal/clusterfeature/clusterfeatureadapter/cluster_service.go

```diff
@@ -37,6 +37,9 @@ type Cluster interface {
 	IsReady() (bool, error)
 	NodePoolExists(nodePoolName string) bool
 	RbacEnabled() bool
+
+	// todo temporarily added operation - remove it when the alternative solution gets in place
+	SetSecurityScan(scan bool)
```

Comment:
How about embedding the base interface in a custom interface in the security scan package?

Reply:
fixed, thx